docs_user/modules/openstack-edpm_adoption.adoc

```diff
@@ -195,6 +197,48 @@ The secret `nova-cell<X>-compute-config` is auto-generated for each
 `cell<X>`. That secret, alongside `nova-migration-ssh-key`, should
 always be specified for each custom `OpenStackDataPlaneService` related to Nova.
 
+* Create a Nova service for libvirt Ceph backend
++
+[source,yaml]
+----
+oc apply -f - <<EOF
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ceph-nova
+  namespace: openstack
+data:
+  03-ceph-nova.conf: |
+    [libvirt]
+    images_type=rbd
+    images_rbd_pool=vms
+    images_rbd_ceph_conf=/etc/ceph/ceph.conf
+    images_rbd_glance_store_name=default_backend
+    images_rbd_glance_copy_poll_interval=15
+    images_rbd_glance_copy_timeout=600
+    rbd_user=openstack
+    rbd_secret_uuid=$CEPH_FSID
+---
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneService
+metadata:
+  name: nova-custom-ceph
```

Comment:
this is for testing only, I'll merge this into nova-compute-extraconfig

Reply:
@gibizer 
```
│ nova-compute-extraconfig-openstack-8gbqd TASK [osp.edpm.edpm_nova : Render nova config files] ***************************                                                             ││ nova-compute-extraconfig-openstack-7b864 TASK [osp.edpm.edpm_container_manage : Create containers managed by Podman for /var/lib/openstack/config/containers] ***                     ││ nova-compute-extraconfig-openstack-7b864 Tuesday 05 March 2024  16:29:04 +0000 (0:00:00.176)       0:03:54.176 *********                                                              ││ nova-compute-extraconfig-openstack-7b864 [WARNING]: ERROR: Container nova_compute exited with code 126 when runed                                                                     ││ nova-compute-extraconfig-openstack-7b864 stderr: time="2024-03-05T16:29:05Z" level=info msg="podman filtering at log                                                                  ││ nova-compute-extraconfig-openstack-7b864 level info" time="2024-03-05T16:29:05Z" level=info msg="Not using native diff                                                                ││ nova-compute-extraconfig-openstack-8gbqd Tuesday 05 March 2024  16:32:19 +0000 (0:00:11.017)       0:02:58.493 *********                                                              ││ nova-compute-extraconfig-openstack-8gbqd ok: [standalone] => (item={'src': 'config.json.j2', 'dest': 'config.json'})                                                                  ││ nova-compute-extraconfig-openstack-8gbqd ok: [standalone] => (item={'src': 'nova-blank.conf', 'dest': 'nova-blank.conf'})                                                             ││ nova-compute-extraconfig-openstack-7b864 for overlay, this may cause degraded performance for building images: kernel                                                                 ││ nova-compute-extraconfig-openstack-7b864 /var/lib/nova/compute_id: operation not permitted                                                                                            ││ nova-compute-extraconfig-openstack-8gbqd ok: [standalone] => (item={'src': 'ssh-config', 'dest': 'ssh-config'})                                                                       ││ nova-compute-extraconfig-openstack-8gbqd ok: [standalone] => (item={'src': '02-nova-host-specific.conf.j2', 'dest': '02-nova-host-specific.conf'})                                    ││ nova-compute-extraconfig-openstack-8gbqd                                                                                                                                              ││ nova-compute-extraconfig-openstack-7b864 fatal: [standalone]: FAILED! => {"changed": false, "msg": "Failed containers: nova_compute"}